doc/t265.md

```diff
@@ -107,3 +107,49 @@ Yes, T265 can work both indoors and outdoors. Just like a person, it can be blin
 ### Do multiple T265 devices interfere with each other?
 
 No, you can use as many T265 devices in a space as you like.
+
+## Appendix
+
+### Wheel odometry calibration file format
+Below is a sample calibration file with inline comments (not a valid file format!) that explain each field. Please note that this is an intermediate solution and a more unified interface will be coming.
+```
+{
+    "velocimeters": [ // array of velocity sensors (currently max. 2 supported)
+        {
+            "scale_and_alignment": [ // 3-by-3 matrix, row-major (multiplies measurement)
+                1.0,
+                0.0000000000000000,
+                0.0000000000000000,
+                0.0000000000000000,
+                1.0,
+                0.0000000000000000,
+                0.0000000000000000,
+                0.0000000000000000,
+                1.0
+            ],
+            "noise_variance": 0.004445126050420168, // measurement covariance (to be determined/tuned)
+            "extrinsics": { // relative transformation of sensor frame w.r.t. T265 body frame
+                "T": [ // translation (in meters)
+                    -0.5059,
+                    -0.6294,
+                    -0.6873
+                ],
+                "T_variance": [ // currently not used, please ignore
+                    9.999999974752427e-7, 
+                    9.999999974752427e-7, 
+                    9.999999974752427e-7
+                ],
+                "W": [ // orientation in axis-angle representation, axis x angle (in rad)
```

Comment:
axis-angle representation? Did you intend for this to Euler angles?

Reply:
This is axis-angle but in the form angle times (unit) axis.